### PR TITLE
Enforce definition of nz or dz in LESdrivenSCM, not both.

### PR DESCRIPTION
**Pull request by ilopezgp**
Enforce definition of nz or dz in LESdrivenSCM, not both. The behavior before this PR accepted a definition of `nz`, but discarded it silently, which is misleading to users.

The current implementation allows defining either nz or dz.

### CODE REVIEW

File: docs/Manifest.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This file is machine-generated - editing it directly is not advised
```
Comment (yairchn) [inline docs/Manifest.toml:2]: you might need to merge an updated main branch

Comment (ilopezgp) [reply docs/Manifest.toml:2]: What do you mean?

Comment (yairchn) [reply docs/Manifest.toml:2]: I am wondering why you have Manifest changes

Comment (ilopezgp) [reply docs/Manifest.toml:2]: Because I used up_deps.jl to keep the dependencies up to date. You can see below that the UpDeps CI check is now passing.

**Comment by ilopezgp** [comment]
I will deal with the mse flag as soon as I get approval.

**Review by charleskawczynski — APPROVED** [review]
Code changes look fine to me. I'd prefer we update deps in separate PRs, just because we can get mse table changes from that alone and doing this helps differentiate changes. That said, I don't want to slow anything down and this looks fine to me.

**Comment by ilopezgp** [comment]
> Code changes look fine to me. I'd prefer we update deps in separate PRs, just because we can get mse table changes from that alone and doing this helps differentiate changes. That said, I don't want to slow anything down and this looks fine to me.

I will take it into account for next time, thanks!

**Pull request merged**